ci/tasks/java-tests.yml

```diff
@@ -0,0 +1,40 @@
+container_limits: {}
```

Comment:
I think I committed this for future use, can swap in for the in-lined yaml task definition once we're happy with it.